nequip/nn/_atomwise.py

```diff
@@ -61,7 +61,7 @@ def __init__(
         self, field: str, out_field: Optional[str] = None, reduce="sum", irreps_in={}
     ):
         super().__init__()
-        assert reduce in ("sum", "mean", "min", "max")
+        assert reduce in ("sum")
```

Comment:
is it necessary to do the assert here? can we rely on the scatter itself to assert?
and why do we abandon "mean", and other modes?